src/components/TargetResourceBrowser.js

```diff
@@ -112,22 +123,27 @@ export default function TargetResourceBrowser() {
           sourceTargetResources
         )
       ) : // No resources exist with the given search term
-      sourceTargetResources &&
-        sourceTargetResources.length === 0 &&
-        sourceSearchValue ? (
-        <div css={[textStyles.body, { marginTop: 10 }]}>
-          No {sourceTarget.readable_name} resources found for search term{" "}
-          {sourceSearchValue}.
+        sourceTargetResources &&
+          sourceTargetResources.length === 0 &&
+          sourceSearchValue ? (
+            <div css={[textStyles.body, { marginTop: 10 }]}>
+              No {sourceTarget.readable_name} resources found for search term{" "}
```

Comment:
This is getting a little out of hand...maybe we can combine some divs and add in a message variable or something?